.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: java
+dist: trusty
```

Comment:
Just because I am curious: What does that mean?

Reply:
We use the "oldstable" build system distro at travis (for the time being), ubuntu "trutsy" instead of "xenial". The new distro version won't suport oracle JDK8 anymore which is for the moment our main target.
https://travis-ci.community/t/install-of-oracle-jdk-8-failing/3038/8